lib/workers/repository/config-migration/pr/index.ts

```diff
@@ -45,33 +46,12 @@ ${
       `JSON5 config file migrated! All comments & trailing commas were removed.`
     : ''
 }
----
-{{#if hasWarningsErrors}}
-{{{warnings}}}
-{{{errors}}}
-{{else}}
-#### Migration completed successfully, No errors or warnings found.
-{{/if}}
----
-
 
-:question: Got questions? Check out Renovate's [Docs](${
-      config.productLinks?.documentation
-    }), particularly the Getting Started section.
-If you need any further assistance then you can also [request help here](${
+:question: Got questions? Something look not right? Please don't hesitate to [request help here](${
```

Comment:
> Something look not right?

Is this proper English? It looks off to me. 😄 I expected something like:

```suggestion
:question: Got questions? Does something look wrong to you? Please don't hesitate to [request help here](${
```